fixtures/contest/irledflat-5x12SIXb.json

```diff
@@ -0,0 +1,614 @@
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [201, 205],
+          "type": "Effect",
+          "effectName": "Macro 10",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [206, 210],
+          "type": "Effect",
+          "effectName": "Macro 11",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [211, 215],
+          "type": "Effect",
+          "effectName": "Macro 12",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [216, 220],
+          "type": "Effect",
+          "effectName": "Macro 13",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [221, 225],
+          "type": "Effect",
+          "effectName": "Macro 14",
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Macro Speed"
+          }
+        },
+        {
+          "dmxRange": [226, 233],
+          "type": "Effect",
+          "effectName": "Music-sensitive Macro 1 (colour sequencing)",
+          "soundControlled": true,
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Mic Sensitivity"
+          }
+        },
+        {
+          "dmxRange": [234, 241],
+          "type": "Effect",
+          "effectName": "Music-sensitive Macro 7 (colour sequencing)",
+          "soundControlled": true,
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Mic Sensitivity"
+          }
+        },
+        {
+          "dmxRange": [242, 249],
+          "type": "Effect",
+          "effectName": "Music-sensitive Macro 11 (colour sequencing)",
+          "soundControlled": true,
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Mic Sensitivity"
+          }
+        },
+        {
+          "dmxRange": [250, 255],
+          "type": "Effect",
+          "effectName": "Music-sensitive Macro 14 (colour sequencing)",
+          "soundControlled": true,
+          "switchChannels": {
+            "Dimmer / Macro Speed / Mic Sensitivity": "Mic Sensitivity"
+          }
+        }
+      ]
+    },
+    "Macro Speed": {
+      "capability": {
+        "type": "EffectSpeed",
+        "speedStart": "slow",
+        "speedEnd": "fast"
+      }
+    },
+    "Mic Sensitivity": {
+      "capability": {
+        "type": "SoundSensitivity",
+        "soundSensitivityStart": "low",
+        "soundSensitivityEnd": "high",
+        "helpWanted": "Is it low…high or high…low?"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "2-channel",
```

Comment:
Can you include the modes' display names please (either as name or shortName)? Those include letters counting from A to G in the manual's mode order, which makes finding them easier.